.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "vendor/ruby-signature"]
```

Comment:
It requires the new Pathname types because the old types did not contain several methods typings.
https://github.com/ruby/ruby-signature/pull/212